stix_shifter/stix_translation/src/modules/bigfix/README.md

```diff
@@ -1,180 +1,337 @@
+```
+### As BigFix is an as-synchronous connector the above translated relevance query is passed as parameter to STIX transmission module
+
+```
+transmit
+"bigfix"
+"{\"host\":\"xx.xx.xx.xx\", \"port\":\"xxxxx\", \"cert_verify\":\"False\"}"
+"{\"auth\":{\"username\":\"xxxxx\",\"password\":\"xxxxxx\"}}"
+query
+"<BESAPI xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:noNamespaceSchemaLocation=\"BESAPI.xsd\"><ClientQuery><ApplicabilityRelevance>true</ApplicabilityRelevance><QueryText>(\"process\", name of it | \"n/a\",  pid of it as string | \"n/a\",  \"sha256\", sha256 of image file of it | \"n/a\",  \"sha1\", sha1 of image file of it | \"n/a\",  \"md5\", md5 of image file of it | \"n/a\",  pathname of image file of it | \"n/a\",  ppid of it as string | \"n/a\",  (if (windows of operating system) then  user of it as string | \"n/a\"  else name of user of it as string | \"n/a\"),  size of image file of it | 0,  (if (windows of operating system) then  (creation time of it | \"01 Jan 1970 00:00:00 +0000\" as time -  \"01 Jan 1970 00:00:00 +0000\" as time)/second else  (start time of it | \"01 Jan 1970 00:00:00 +0000\" as time -  \"01 Jan 1970 00:00:00 +0000\" as time)/second))  of processes whose (((name of it as lowercase = \"dnsmasq\" as lowercase)) AND (if (windows of operating system) then (creation time of it | \"01 Jan 1970 00:00:00 +0000\" as time is greater than or equal to \"10 Jan 2013 08:43:10 +0000\" as time AND creation time of it | \"01 Jan 1970 00:00:00 +0000\" as time is less than or equal to \"23 Oct 2019 10:43:10 +0000\" as time) else (start time of it | \"01 Jan 1970 00:00:00 +0000\" as time is greater than or equal to \"10 Jan 2013 08:43:10 +0000\" as time AND start time of it | \"01 Jan 1970 00:00:00 +0000\" as time is less than or equal to \"23 Oct 2019 10:43:10 +0000\" as time)))</QueryText><Target><CustomRelevance>true</CustomRelevance></Target></ClientQuery></BESAPI>"
+
+```
+### A search_id is returned which is further passed to STIX transmission result module 
+```
+{'search_id': '4192', 'success': True}
+```
+### Transmit get result
+```
+transmit bigfix "{\"host\":\"xx.xx.xx.xx\", \"port\":\"xxxxx\", \"cert_verify\":\"False\"}" {\"auth\":{\"username\":\"hclapi\",\"password\":\"jnGfdEYY8T$e\"}} results 4192 <offset> <length> 
```

Comment:
please sanitize the username and password. put something random.

Reply:
Incorporated with latest Commit.